winvnc/winvnc/winvnc.rc

```diff
@@ -504,17 +532,23 @@ BEGIN
     BEGIN
         BLOCK "000004b0"
         BEGIN
-            VALUE "Comments", "UltraVNC - Remote Control for all"
-            VALUE "CompanyName", "UltraVNC"
-            VALUE "FileDescription", "VNC server"
-            VALUE "FileVersion", "2024.02.17.3"
+#if isRC == 1
+#include "buildtime.h" // Defines macro BUILDTIME
+            VALUE "Buildtime", BUILDTIME
+#endif
+            VALUE "Comments", "UltraVNC Server"
+            VALUE "CompanyName", STR_COMPANYNAME
+            VALUE "FileDescription", "UltraVNC Server"
+            VALUE "FileVersion", STR_FILEVERSION
             VALUE "InternalName", "WinVNC"
-            VALUE "LegalCopyright", "Copyright � 2021 UltraVNC"
-            VALUE "LegalTrademarks", "VNC"
+            VALUE "LegalCopyright", STR_COPYRIGHT
             VALUE "OriginalFilename", "WinVNC.exe"
-            VALUE "PrivateBuild", "1.4.4.0"
-            VALUE "ProductName", "UltraVNC"
-            VALUE "ProductVersion", "2024.02.17.3"
+            VALUE "PrivateBuild", ""
+            VALUE "ProductName", "UltraVNC Server"
+            VALUE "ProductVersion", STR_PRODUCTVERSION
+#if isRC == 1
+			VALUE "SpecialBuild", STR_SPECIALBUILD

```

Comment:
Don't know the impact of changing productname and comapny name.
This is another software with the same filename.